### DISCUSSION TITLE
How do I force SuperSlicer to print 3 perimeters for a set 1.44mm wall thickness?

### DISCUSSION THREAD

**Discussion by briggs2025** [supermerill/SuperSlicer#2278] [Q&A]
I designed a test part to print with 3 perimeters, not 2, or 4 or any other number. The part has a uniform 1.44mm wall thickness (that's 3 perimeters times .48mm extrusion width, using an 0.40mm nozzle). My layer height is .24mm.  No matter what I change the extrusion width and spacing to I only get 2 perimeters. In S3D, Ideamaker, and Cura I spec 3 perimeters at .48mm extrusion width and it gives me 3 perimeters.

How do I figure out the width and spacing to force SuperSlicer to give me 3 perimeters? Thank you!!

**Comment by supermerill** [comment]
An odd perimeter number is only possible if the middle one is really a circle/loop, or the gapfill won't be categorized as "perimeter".
If you want the middle extrusion to be the same "width" as the other ones, it's a bit difficult to compute.
What I advise you is to just use spacing. Something like 0.49 for the external perimeter 'spacing+width' and 0.46 for perimeter spacing.

**Reply by briggs2025** [reply]
Hi Supermerill, thanks for the reply. Yes all three perimeters are loops. If I turn on gap filling it will show what basically looks like a middle loop. If I keep gap filling off it only shows two perimeters, no matter what I change the spacing values to.  I think there is a need to being able to simply spec the extrusion width you're looking for (like .48mm) and the number of perimeters you want, and then let the software calculate the rest, you know? So much of what I design is thin wall parts. I'm also thinking about downloaded parts that say, "use 4 perimeters" for the optimal strength/weight/whatever, etc.

**Reply by supermerill** [reply]
That's what the spacing column is in the width section.
I just test and it may need a little more tolerance to not be turned into gapfill, so 0.49 and 0.45, as 0.49*2+ 0.45 = 1.43mm
![image](https://user-images.githubusercontent.com/6536403/151415551-ff7a03f4-e98f-4de8-a7e6-05c4c055cb6d.png)

**Reply by briggs2025** [reply]
What are your width settings at? Thank you.

**Reply by supermerill** [reply]
not relevant, as the spacing is set. The width is computed automatically.

**Reply by briggs2025** [reply]
![image](https://user-images.githubusercontent.com/41799646/151444103-5edec8d6-6bd3-4001-b049-38a7a324fec0.png)


 I just can't seem to get this! I change the spacing to what you have but it doesn't work, I still only get two perimeters. I think it would be helpful to see your widths.

**Reply by supermerill** [reply]
you have to activate gapfill, as the algorithm can't output an asymmetric number of perimeter with gapfill deactivated
[1.44wallcube.3mf.zip](https://github.com/supermerill/SuperSlicer/files/7953987/1.44wallcube.3mf.zip)

**Reply by briggs2025** [reply]
Yes enabling gap fill does the trick. Thanks Supermerill.
